src/lib/index.js

```diff
@@ -8,6 +8,7 @@ import Circos from './components/Circos.react';
 import OncoPrint from './components/OncoPrint.react';
 import AlignmentChart from './components/AlignmentChart.react';
 import AlignmentViewer from './components/AlignmentViewer.react';
+import Molecule2dViewer from './components/Molecule2dViewer.react';
```

Comment:
I thought we once reordered these imports alphabetically... As the number of components increase, it will be helpful to read them in alphabetical order. Could you please reorder both imports and exports?

Reply:
Fixed in fa6f132 and 8add7f7!